### PR TITLE
Update py/embedded-server setup for PyPi

### PR DESCRIPTION
**Pull request by jjbrosnan**
This PR updates the setup.py file to use a new README for PyPi, which contains the basic installation instructions above the dev environment instructions.

### CODE REVIEW

File: py/embedded-server/README_PyPi.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Deephaven Python Embedded Server
```
Comment (devinrsmith) [inline py/embedded-server/README_PyPi.md:1]: I would have the header be `deephaven-server` to match the PyPi package name.

File: py/embedded-server/README_PyPi.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+### Build
+From the root directory of this repository
+```shell
+$ ./gradlew :py-embedded-server:assemble
+```
+
+### Install
+```shell
+$ pip install py/embedded-server/build/wheel/deephaven_server-0.12.0-py3-none-any.whl
+```
```
Comment (devinrsmith) [inline py/embedded-server/README_PyPi.md:30]: IMO, we should ax all of this - these are development notes and can go elsewhere, but they don't belong on PyPi.

File: py/embedded-server/README_PyPi.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+## Quick start
+
```
Comment (devinrsmith) [inline py/embedded-server/README_PyPi.md:33]: We should have a note that Java 11+ is required, and `JAVA_HOME` should be set.

**Review by devinrsmith — CHANGES_REQUESTED** [review]

File: py/embedded-server/README_PyPi.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+# Deephaven Python Embedded Server
+
+Embeds the Deephaven Core server into a python module, so that it can be started from python, with tables produced
+directly as part of startup.
```
Comment (devinrsmith) [inline py/embedded-server/README_PyPi.md:4]: I think this should be a more python-end-user introduction - we shouldn't be focused on the "embeds" part of it. A link to deephaven-core is fine.

File: py/embedded-server/README_PyPi.md (hunk 3 above)
Comment (jjbrosnan) [reply py/embedded-server/README_PyPi.md:33]: Is the first line in the setup section not sufficient?

**Review by margaretkennedy — APPROVED** [review]

File: py/embedded-server/README_PyPi.md (hunk 3 above)
Comment (margaretkennedy) [reply py/embedded-server/README_PyPi.md:19]: The line seems enough to me.

File: py/embedded-server/README_PyPi.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# deephaven-server
+
+The Dockerless version of [deephaven-core](https://www.github.com/deephaven/deephaven-core).
+
+Deephaven Community Core is a real-time, time-series, column-oriented analytics engine with relational database features. It is an open version of [Deephaven Enterprise](https://deephaven.io/enterprise/), which serves as the data backbone for prominent hedge funds, banks, and financial exchanges.
+
+`deephaven-server` enables users to run Deephaven Community Core without Docker.
```
Comment (devinrsmith) [inline py/embedded-server/README_PyPi.md:7]: I'm not a fan of explicitly calling our main product a "docker" product; and thus, `deephaven-server` is "Deephaven minus docker". I think highlighting that this is a python-native way of starting Deephaven is what we should strive for conveying.

File: py/embedded-server/README_PyPi.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+## Setup
+
+Java 11+ is required for this module, and the `JAVA_HOME` environment variable must be set appropriately.  Once that's done, you can use pip:
```
Comment (devinrsmith) [inline py/embedded-server/README_PyPi.md:11]: I don't like attaching this as a *setup* requirement - it's not necessary for pip installs. That said, it is important info, and having it set during pip install won't hurt.

I'd suggest creating a "## Requirements" header before Setup; where we can also mention that it requires Python 3.7+.

**Review by devinrsmith — CHANGES_REQUESTED** [review]

**Review by devinrsmith — APPROVED** [review]

**Review by devinrsmith — APPROVED** [review]

**Pull request merged**